File: Python/card.py
from payment import Payment

class Card(Payment):
    Number= int
    Cvv= int 
    Data=str
    def __init__(self,id,Number,Cvv,Data):
        super().__init__(id)
        self.Number=Number
        self.Data=Data
        self.Cvv=Cvv
        

File: Java/Card.java
class Card extends Payment{
    String Number;
    Integer Cvv;
    String Data;

    public Card(Integer id,
    String Number,Integer Cvv,String Data){
        super(id);
        this.Number=Number;
        this.Cvv=Cvv;
        this.Data=Data;
        
    }
}
